cmd/serverless/main.go

```diff
@@ -321,6 +278,54 @@ func runAgent(stopCh chan struct{}) (serverlessDaemon *daemon.Daemon, err error)
 	return
 }
 
+func startTraceAgent(wg *sync.WaitGroup, lambdaSpanChan chan *pb.Span, coldStartSpanId uint64, serverlessDaemon *daemon.Daemon) {
+	// starts trace agent
+	defer wg.Done()
+	traceAgent := &trace.ServerlessTraceAgent{}
+	traceAgent.Start(config.Datadog.GetBool("apm_config.enabled"), &trace.LoadConfig{Path: datadogConfigPath}, lambdaSpanChan, coldStartSpanId)
+	serverlessDaemon.SetTraceAgent(traceAgent)
+}
+
+func startAppSec() invocationlifecycle.InvocationSubProcessor {
+	var httpsecSubProcessor invocationlifecycle.InvocationSubProcessor
+	appsecApi, err := appsec.New()
+	if err != nil {
+		log.Error("appsec: could not start: ", err)
+	} else if appsecApi != nil {
+		log.Info("appsec: started successfully")
+		httpsecSubProcessor = httpsec.NewInvocationSubProcessor(appsecApi) // note that the receiving variable is in the parent scope
+	}
+	return httpsecSubProcessor
+}
+
+func enableTelemetryCollection(wg *sync.WaitGroup, serverlessID registration.ID, logChannel chan *logConfig.ChannelMessage) {
+	defer wg.Done()
+	if len(os.Getenv("DD_LOCAL_TEST")) > 0 {
```

Comment:
The string `"DD_LOCAL_TEST"` is available via the const `daemon.localTestEnvVar`. You would need to export the const though first.

Reply:
good idea!